pkg/validations/createvalidations/gitops.go

```diff
@@ -23,5 +23,18 @@ func ValidateGitOpsNameIsUnique(ctx context.Context, k validations.KubectlClient
 	if len(existingGitOps) > 0 {
 		return fmt.Errorf("gitOpsConfig %s already exists", spec.Spec.GitOpsRef.Name)
 	}
+
+	mgmtCluster, err := k.GetEksaCluster(ctx, cluster, cluster.Name)
+	if err != nil {
+		return err
```

Comment:
nit: do we want to have additional details here for the error message?

Reply:
it is sorta taken care of in the executable code